Firestore/core/src/firebase/firestore/immutable/array_sorted_map.h

```diff
@@ -110,7 +110,7 @@ class FixedArray {
     return begin() + size_;
   }
 
-  array_type contents_;
+  array_type contents_{};
```

Comment:
Note: because `FixedArray` has a default constructor, the previous and the current versions are equivalent.

Reply:
Oops. Sorry, got carried away.